DCHP/download.html

```diff
@@ -0,0 +1,82 @@
+            crossorigin="anonymous"
+        >
+        <link rel="stylesheet" href="../CSS/StyleSheet.css">
+        <script src="https://kit.fontawesome.com/f7a97ee3aa.js" crossorigin="anonymous"></script>
+        <!-- Website Favicon -->
+        <link rel="shortcut icon" type="image/png" href="../Images/Logo.png">
+        <!-- Logo Font -->
+        <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@800&display=swap" rel="stylesheet">
+        <title>DCHP: Download</title>
+    </head>
+    <body>
+        <!-- Navigation Bar Start -->
+        <nav class="navbar navbar-expand-sm">
+            <!-- Brand -->
+            <a class="navbar-brand devsoc-nav-logo" href="index.html">[}EVSOC_</a>
+            <!-- Toggler/collapsibe Button -->
+            <button
+                class="navbar-toggler"
+                type="button"
+                data-toggle="collapse"
+                data-target=".navbar-collapse"
+            >
+                <span class="navbar-toggler-icon">
+                    <i class="fa fa-bars" style="color: #fff; font-size: 28px"></i>
+                </span>
+            </button>
+            <div class="collapse navbar-collapse" id="collapsibleNavbar">
+                <ul class="navbar-nav ml-auto">
+                    <!-- Page -->
+                    <li class="nav-item">
+                        <a class="nav-link devsoc-nav-item" href="home.html">
+                            <i class="fa fa-fw fa-home" style="color: #fff"></i>
+                            Home
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link devsoc-nav-item" target="_blank" href="https://discord.com/invite/JrPpeWQ">
+                            <i class="fa fa-fw fa-user-friends" style="color: #fff"></i>
+                            Join
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link devsoc-nav-item" href="mailto:devsoc@ntu.ac.uk">
+                            <i class="fa fa-fw fa-envelope" style="color: #fff"></i>
+                            Contact
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link devsoc-nav-item" href="Archive/archive.html">
+                            <i class="fa fa-fw fa-archive" style="color: #fff"></i>
+                            Archive
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <div class="container-fluid h-100">
+            <div class="row my-5 justify-content-center align-items-center">
+                <iframe
+                    width="1120"
+                    height="630"
+                    src="https://www.youtube.com/embed/dQw4w9WgXcQ"
```

Comment:
autoplay query needs to be added to trigger autoplay. but may not work on some browsers or if certain conditions aren't met
```suggestion
                    src="https://www.youtube.com/embed/dQw4w9WgXcQ?autoplay=1"
```